doc/modeling/indexes.md

```diff
@@ -9,8 +9,8 @@ Since version 11, PostgreSQL supports [covering indexes](https://paquier.xyz/pos
 ```c#
 protected override void OnModelCreating(ModelBuilder modelBuilder)
     => modelBuilder.Entity<Blog>()
-                   .ForNpgsqlHasIndex(b => b.Id)
-                   .ForNpgsqlInclude(b => b.Name);
+                   .HasIndex(b => b.Id)
+                   .ForNpgsqlInclude(nameof(b.Name));
```

Comment:
Why not use the expression-based overload?

https://github.com/npgsql/Npgsql.EntityFrameworkCore.PostgreSQL/blob/10802aef53e3f61b9cc068986b52ccbfea630a84/src/EFCore.PG/Extensions/NpgsqlIndexBuilderExtensions.cs#L100-L102